### PR TITLE
fix: add some error message translations

### LINKED ISSUE #2124: Some error messages are not internationalized
### Is your feature request related to a problem? Please describe.

Should these error messages be added to add internationalization?

image

### Describe the solution you'd like

If these error messages need to be internationalized, I can submit a PR.

### PR DESCRIPTION
**Pull request by ChaominRuan**
Closes #2124

### CODE REVIEW

File: lib/features/modeling/cmd/UpdateModdlePropertiesHandler.js [hunk 1]
```diff
@@ -22,21 +22,27 @@ import {
...
 
   var element = context.element,
       moddleElement = context.moddleElement,
-      properties = context.properties;
+      properties = context.properties,
+      translate = this._translate;
 
   if (!moddleElement) {
-    throw new Error('<moddleElement> required');
+    throw new Error(translate('<moddleElement> required'));
```
Comment (nikku) [inline lib/features/modeling/cmd/UpdateModdlePropertiesHandler.js:45]: Are you sure you want to translate technical error messages? These are not meant to appear to the end user in any way, but log to the console.

File: test/spec/ModelerSpec.js [hunk 2]
```diff
@@ -205,14 +205,17 @@ describe('Modeler', function() {
           'already rendered {element}',
           'correcting missing bpmnElement on {plane} to {rootElement}',
           'diagram not part of bpmn:Definitions',
-          'element {element} referenced by {referenced}#{property} not yet drawn',
           'element required',
+          'element {element} referenced by {referenced}#{property} not yet drawn',
           'failed to import {element}',
+          'flow elements must be children of pools/participants',
```
Comment (nikku) [inline test/spec/ModelerSpec.js:211]: Let's ensure this is being captured when generating the translations JSON, executed via 

```
npm run collect-translations
```

File: lib/features/modeling/cmd/UpdateModdlePropertiesHandler.js (hunk 1 above)
Comment (ChaominRuan) [reply lib/features/modeling/cmd/UpdateModdlePropertiesHandler.js:45]: I see a lot of throw new Errors with translations, for example: [    throw new Error(translate('element required'));](https://github.com/bpmn-io/bpmn-js/blob/5d1c447e87e398327845dce14b3319187845c63b/lib/features/modeling/cmd/UpdatePropertiesHandler.js#L91)

Comment (nikku) [reply lib/features/modeling/cmd/UpdateModdlePropertiesHandler.js:45]: You're right. I'll get this to the team to discuss.

Comment (nikku) [reply lib/features/modeling/cmd/UpdateModdlePropertiesHandler.js:45]: Some work in progress: We consider removing translation of technical errors - https://github.com/bpmn-io/bpmn-js/pull/2145.

**Comment by nikku** [comment]
@ChaominRuan as per prior discussion we're removing the translation of technical errors - https://github.com/bpmn-io/bpmn-js/pull/2145. 

This will benefit users that search for the root cause of an issue in _one_ language, and find respective (official) resources.

**Comment by nikku** [comment]
Thanks again for your contribution, and for allowing us to identify this issue. I'll close this PR as _won't fix_ as your changes only touch upon the technical errors that we no longer translate (cf. https://github.com/bpmn-io/bpmn-js/pull/2137#issuecomment-2074367305).

File: lib/features/modeling/cmd/UpdateModdlePropertiesHandler.js (hunk 1 above)
Comment (nikku) [reply lib/features/modeling/cmd/UpdateModdlePropertiesHandler.js:45]: Decided (cf. https://github.com/bpmn-io/bpmn-js/pull/2145) - We're removing translation of technical errors.

**Pull request closed without merging**